### PR TITLE
fix: TextArea wrapper with div when `!showCount`

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix TextArea wrapped with additional div when `showCount` is `false`. And pass `className` & `style` to wrapper when `showCount` is `true`.          |
| 🇨🇳 Chinese |   修复 TextArea 没有设置 `showCount` 时仍然会包裹 div 的问题，同时解决 `showCount` 下 `className` 和 `style` 没有传递给最外层节点的问题。       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
https://lgtm.com/projects/g/ant-design/ant-design/snapshot/0087b32869d11af7769bcbe4938d67d3a73db365/files/components/input/TextArea.tsx?sort=name&dir=ASC&mode=heatmap#xb5c05dc21d3a94f1:1

**Review by afc163 — COMMENTED** [review]
~

**Review by afc163 — CHANGES_REQUESTED** [review]
https://lgtm.com/projects/g/ant-design/ant-design/snapshot/0087b32869d11af7769bcbe4938d67d3a73db365/files/components/input/TextArea.tsx?sort=name&dir=ASC&mode=heatmap#xb5c05dc21d3a94f1:1

File: components/input/TextArea.tsx [hunk 1]
```diff
@@ -100,35 +101,56 @@ class TextArea extends React.Component<TextAreaProps, TextAreaState> {
...
+    if (showCount) {
+      const valueLength = [...value].length;
+      const dataCount = `${valueLength}${hasMaxLength ? ` / ${maxLength}` : ''}`;
+
+      textareaNode = (
+        <div
+          className={classNames(
+            `${prefixCls}-textarea`,
+            {
+              [`${prefixCls}-textarea-show-count`]: showCount,
```
Comment (afc163) [inline components/input/TextArea.tsx:140]: 这里也不需要判断了。

**Review by afc163 — APPROVED** [review]

**Review by xrkffgg — APPROVED** [review]

**Pull request merged**